test/components/__snapshots__/Bar.spec.jsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Bar should display the Searchbar 1`] = `
 <div
-  className="coz-bar-wrapper"
+  className="coz-bar-wrapper theme-undefined"
```

Comment:
@CPatchane you understand why it's `undefined`?

Reply:
Because we don't test against connected components but we used the dumb component https://github.com/cozy/cozy-bar/blob/master/test/components/Bar.spec.jsx#L2, you have to add a new props and check props change in the unit tests
Could you also add tests for this new API method? It should the same than [for `setLocale`](https://github.com/cozy/cozy-bar/blob/master/test/lib/api.spec.js#L104-L112)